api/apps/articles/index.coffee

```diff
@@ -7,5 +7,5 @@ app = module.exports = express()
 app.get '/articles', setUser, routes.index
 app.get '/articles/:id', routes.find, routes.show
 app.post '/articles', setUser, authenticated, routes.restrictFeature, routes.create
-app.put '/articles/:id', setUser, authenticated, routes.find, routes.restrictFeature, routes.update
+app.put '/articles/:id', setUser, authenticated, routes.restrictFeature, routes.update
```

Comment:
We no longer `find` the article first since
1. We already `find` once within `save`
2. For PUT, we don't want to merge an existing article + input _before_ Joi.validate because we'd have to `.toString()` all the `ObjectIds` again. Before, the merge was helpful because it overrode defaults, but now there is an option to exclude that. Ends up being much cleaner.